cg/utils/string.py

```diff
@@ -0,0 +1,10 @@
+"""Utils related to string manipulation."""
+
+from cg.exc import CgError
+
+
+def get_element_from_split(value: str, element_position: int, split: str) -> str:
+    elements: list[str] = value.split(split)
+    if len(elements) < element_position:
+        raise CgError(message="Provided element position out of bounds.")
+    return elements[element_position]
```

Comment:
Same for this one. `some_string.split("-")[1] ` is quite basic python syntax and I don't think it needs to be hitten away in a function like this

Reply:
removed this function in #3575